### PR TITLE
Move article list actions to a (vertical) 3-dot more icon menu #103

### PR DESCRIPTION
**Pull request by dominklenda**
Improvement in author page improvements #103 

move article list actions to a (vertical) 3-dot more icon menu, horizontally aligned w/ Filter articles box (see mockup; only displayed for users w/ proper permissions for these actions, i.e., admins or users own author page)
- for "LINK EXISTING ARTICLE" menu item, it's OK that article selector component opens just below menu item wherever it happens to be (just like how it currently opens below the "LINK EXISTING ARTICLE" button)
- ensure proper (vertical) spacing between Filter articles box and author info URLs (this applies to non-logged-in users too)

### CODE REVIEW

**Comment by eplebel** [comment]
good start, but (1) the LINK EXISTING ARTICLE action doesn't appear to work (clicking on it does nothing) and (2) lots of stylistic stuff not conforming to specified specs, so please look at specs more carefully (e.g., vertical space between social links & article list, action item text capitalization & ICONS, missing info icon, etc.) 

see specs: https://user-images.githubusercontent.com/4512335/76425370-e445d680-637f-11ea-8783-245705b53fc5.png
https://github.com/ScienceCommons/curate_science/issues/103

File: src/pages/AuthorPage.jsx [hunk 1]
```diff
@@ -77,7 +79,14 @@ const styles = theme => ({
...
             visibility: 'visible'
         }
-    }
+    },
+  actionMenu: {
+      paddingTop: theme.spacing(1),
+      paddingBottom: theme.spacing(1),
+      margin: 0,
+      position: 'absolute',
+      right: (C.COL_WIDTH - C.CARD_COL_WIDTH)/2,
```
Comment (eplebel) [inline src/pages/AuthorPage.jsx:88]: please replace following 3 lines
```
margin: 0,
position: 'absolute',
right: (C.COL_WIDTH - C.CARD_COL_WIDTH)/2,
 ```
with 
`float:right`
to fix horizontal alignment issue

image

**Comment by eplebel** [comment]
also, menu does NOT close correctly after clicking a menu item (e.g., click ADD ARTICLE, add new article & SAVE, and notice that upon return to the author page, the menu is STILL OPEN)

**Comment by dominklenda** [comment]
Yes, that's a bit messy. I made some fixes inside AuthorPage.jsx, but I think the best option would be to prepare a whole new reusable component - AuthorPageActions.jsx.

**Comment by eplebel** [comment]
ok thanks for the update.  

we can give it another try, but if we think this may be overly ambitious, we could just put a hold on it, and work on the next issue priority issue i mentioned in my email (display full-text on page load for the article page).

**Comment by dominklenda** [comment]
@eplebel thanks for all these comments! i tried to address them in the #133. please let me know if any other issues appear.

**Pull request merged**